bcrs-ui/src/App.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div id="app">
+    <div class="app-body">
+      <main>
+        <router-view />
+      </main>
+    </div>
+
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import HelloWorld from './components/HelloWorld.vue'
+
+@Component({
+  components: {
+    HelloWorld
+  }
+})
+export default class App extends Vue {}
+</script>
+
+<style lang="scss">
+#app {
+  font-family: 'Avenir', Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+  margin-top: 60px;
+}
```

Comment:
Some of this is obsolete/invalid for us (like font). See if you can use our existing classes and get rid of CSS here.

Reply:
It's definitely unrelated and obsolete to the project as a whole. It's just some styling for the VUE Splash page which will be removed. In that context, would you still see some value in removing this and restyling the splash page?

Reply:
I guess it doesn't really matter either way. Leave the code as-is for now.